tests/robot-cases/Group1-Nightly/OIDC.robot

```diff
@@ -123,3 +123,25 @@ Test Case - OIDC Group User
     ${output}=  Run Keyword And Ignore Error  Switch To Configure
     Should Be Equal As Strings  '${output[0]}'  'FAIL'
     Close Browser
+
+
+Test Case - Delete an OIDC User
+    Init Chrome Driver
+    # sign in with admin role
+    ${admin_user}=  Set Variable  admin_user
+    ${admin_pwd}=  Set Variable  zhu88jie
+    Sign In Harbor With OIDC User  ${HARBOR_URL}  username=${admin_user}  password=${admin_pwd}  login_with_provider=ldap
+    Switch to User Tag
+    # shoule be able to delete an OIDC user
+    Retry Element Click   xpath=//clr-dg-row[contains(.,'test7')]//div[contains(@class,'clr-checkbox-wrapper')]//label[contains(@class,'clr-control-label')]
```

Comment:
Move locators value to OIDC_Elements.robot instead of showing in test case file.

Reply:
done